dev-docs/bidders/insticator.md

```diff
@@ -24,12 +24,128 @@ sidebarType: 1
+| `video.maxextended`    | optional    | Max extended ad duration beyond the maxduration if extension is allowed. Blank or 0 - blocked. -1 - allowed without time limit | `30` |
+| `video.minbitrate`     | optional    | Minimum bit rate in Kbps                                        | `5`                           |
+| `video.maxbitrate`     | optional    | Maximum bit rate in Kbps                                        | `10000`                           |
+| `video.playbackend`    | optional    | The event that causes playback to end. (see OpenRTB v2.5 section 5.11 for options)  | `1` |
+| `video.delivery`       | optional    | Supported delivery methods (1 = streaming, 2 = progressive, 3 = download). If none specified, assume all are supported. | `[1, 2]`                      |
+| `video.pos`            | optional    | Ad position on screen. (see OpenRTB v2.5 section 5.4 for options)     | `1`                           |
+| `video.api`            | optional    | List of supported API frameworks for this impression. Refer to [API Frameworks](#api-frameworks) | `[2, 7]`                   |
+
+### Example
+
+```javascript
+var adUnits = [
+    {
+        code: 'insticator-video-ad-2',
+        mediaTypes: {
+            video: {
+                playerSize: [640, 480],
+                w: 640,
+                h: 480,
+                mimes: ['video/mp4', 'video/x-flv'],
+                protocols: [2, 3, 5, 6, 7, 8],
+                placement: 3,
+                startdelay: 0,
+                api: [2, 7],
+                skip: 0,
+                minduration: 1,
+                maxduration: 30,
+                playbackmethod: [1, 3],
+                battr: [13, 14],
+                linearity: 1,
+                minbitrate: 10,
+                maxbitrate: 10000
+            }
+        },
+        bids: [{
+            bidder: 'insticator',
+            params: {
+                adUnitId: 'example_adunit_id'
+            }
+        }],
+        ...
+    }];
+```
+
+### Video Protocols
+
+{: .table .table-bordered .table-striped }
+| Value | Description                                              |
+|-------|----------------------------------------------------------|
+| `2`   | VAST 2.0                                                 |
+| `3`   | VAST 3.0                                                 |
+| `5`   | VAST 2.0 Wrapper                                         |
+| `6`   | VAST 3.0 Wrapper                                         |
+| `7`   | VAST 4.0                                                 |
+| `8`   | VAST 4.0 Wrapper                                         |
+
+#### API Frameworks
+
+{: .table .table-bordered .table-striped }
+| Value | Description |
+|-------|-------------|
+| `1`   | VPAID 1.0   |
+| `2`   | VPAID 2.0   |
+| `3`   | MRAID-1     |
+| `4`   | ORMMA       |
+| `5`   | MRAID-2     |
+| `6`   | MRAID-3     |
+| `7`   | OMID-1      |
```

Comment:
This can also be removed and replaced with a link to the OpenRTB spec

Reply:
I will remove API Frameworks but for Protocols we don't support all protocols mentioned in ORTB spec, so we are mentioning only the supported types

Reply:
@muuki88  So do I need to remove Protocols also? as it contains specific protocols which we use.

Reply:
Is there any protocol that's not defined in 5.8 protocols in the spec https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf ?

You can reference the Link above and the section number for the things you support.

Reply:
@muuki88 I have updated the docs, can you please verify